Functions.Templates/Templates/DurableFunctionsOrchestrator-CSharp/build.config/template.json

```diff
@@ -0,0 +1,58 @@
+﻿{
+  "author": "Microsoft",
+  "classifications": [
+    "Azure Function"
+  ],
+  "name": "DurableFunctionsOrchestratorCSharp",
+  "identity": "Azure.Function.CSharp.DurableFunctionsOrchestratorTrigger",
+  "groupIdentity": "Azure.Function.DurableFunctionsOrchestratorTrigger",
+  "shortName": "EventHub",
+  "tags": {
+    "language": "C#",
+    "type": "item"
+  },
+  "sourceName": "DurableFunctionsOrchestratorCSharp",
+  "symbols": {
+    "namespace": {
+      "description": "namespace for the generated code",
+      "replaces": "Company.Function",
+      "type": "parameter"
+    }
+  },
+  "primaryOutputs": [
+    {
+      "path": "EventHubTriggerCSharp.cs"
```

Comment:
>EventHubTriggerCSharp.cs [](start = 15, length = 24)

Should EventHubTriggerCSharp.cs be changed to something else, like OrchestrationTriggerCSharp.cs?